roles/github/templates/build-kayobe-docker-image.yml.j2

```diff
@@ -9,6 +14,11 @@ env:
 
 jobs:
   %% format_file_name(workflow.file_name) %%:
+    <%- if github_environment_type == 'kayobe' +%>
+    strategy:
+      matrix:
+        environment: %% github_kayobe_environments %%
+    <%- endif +%>
     runs-on: %% github_runs_on %%
     container:
       image: docker:24.0-git
```

Comment:
I noticed that permissions.packages gets rendered as `none` in the tests even with the default registry.

Reply:
This has been resolved.